src/main/java/net/pms/util/ProcessUtil.java

```diff
@@ -266,14 +293,14 @@ public static void reboot(ArrayList<String> cmd, Map<String,String> env, String
 			startdir = System.getProperty("user.dir");
 		}
 
-		System.out.println("starting: " + StringUtils.join(cmd, " "));
+		System.out.println("Starting: " + StringUtils.join(cmd, " "));
 
 		final ProcessBuilder pb = new ProcessBuilder(cmd);
 		if (env != null) {
 			pb.environment().putAll(env);
 		}
 		pb.directory(new File(startdir));
-		System.out.println("in directory: " + pb.directory());
+		System.out.println("In folder: " + pb.directory());
```

Comment:
They are synonyms, except that "folder" is somewhat broader. A folder can e.g be a "virtual" folder, while i directory per definition is tied to a physical disk directory. The reason I prefer folder is because I think it is more descriptive and that "directory" is a "dinosaur" from the DOS/terminal era.

It really doesn't matter though, my main point in editing it was to capitalize the "I".